spec/support/shared_examples/features/admin_content_nodes_spec.rb

```diff
@@ -0,0 +1,171 @@
+require 'rails_helper'
+
+RSpec.shared_examples :admin_content_node do
+  let(:node_button) { node_name.to_s.humanize }
+  let!(:subject_node) { FactoryGirl.create node_name, author: account }
+  let!(:blocked_node) { FactoryGirl.create "blocked_#{node_name}", author: account }
+
+  before do
+    another_account = FactoryGirl.create :confirmed_account
+    FactoryGirl.create node_name, author: another_account
+
+    login_with(account)
```

Comment:
doesn't see any sense to repeat this before every test. It' necessary only once, for others we can authorize user manually to speed up specs